pkgs/tools/misc/spigot/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, buildPackages
+, fetchgit
+, autoreconfHook
+, gmp
+, ncurses
+, halibut
+, perl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "spigot";
+  version = "b1b0b20";
```

Comment:
That's a bit misleading, I think. This is the `rev`, not the version. The version should still be `20200101`.